staging/vhost-device-sound/src/device.rs

```diff
@@ -54,15 +54,16 @@ impl VhostUserSoundThread {
     ) -> Result<Self> {
         queue_indexes.sort();
 
-        Ok(Self {
+        let thread = VhostUserSoundThread {
```

Comment:
```suggestion
        let thread = Self {
```

Clippy would complain about this (not using `Self`)

Reply:
I think that was a mistake, I don't remember why I changed that. returning Ok(self{}) is fine.